src/create-model.jl

```diff
@@ -1018,59 +1018,87 @@ function create_model(
             accumulated_initial_units[a in A, y in Y],
             sum(values(graph[a].initial_units[y]))
         )
-        @expression(
+
+        ### Expressions for multi-year investment simple method
+        accumulated_investment_units_using_simple_method = @expression(
             model,
-            accumulated_units_simple_method[y ∈ Y, a ∈ decommissionable_assets_using_simple_method],
-            accumulated_initial_units[a, y] + sum(
+            accumulated_investment_units_using_simple_method[
+                a ∈ decommissionable_assets_using_simple_method,
+                y in Y,
+            ],
+            sum(
                 assets_investment[yy, a] for
                 yy in Y if a ∈ investable_assets_using_simple_method[yy] &&
                 starting_year_using_simple_method[(y, a)] ≤ yy ≤ y
-            ) - sum(
+            )
+        )
+        accumulated_decommission_units_using_simple_method = @expression(
+            model,
+            accumulated_decommission_units_using_simple_method[
+                a ∈ decommissionable_assets_using_simple_method,
+                y in Y,
+            ],
+            sum(
                 assets_decommission_simple_method[yy, a] for
                 yy in Y if starting_year_using_simple_method[(y, a)] ≤ yy ≤ y
             )
         )
+        @expression(
+            model,
+            accumulated_units_simple_method[a ∈ decommissionable_assets_using_simple_method, y ∈ Y],
+            accumulated_initial_units[a, y] +
+            accumulated_investment_units_using_simple_method[a, y] -
+            accumulated_decommission_units_using_simple_method[a, y]
+        )
+
+        ### Expressions for multi-year investment compact method
+        accumulated_decommission_units_using_compact_method = @expression(
```

Comment:
```suggestion
        @expression(
```
Since you're naming this one, you don't need the atribution.